src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/CatalogPromotion.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources
+    xmlns="https://api-platform.com/schema/metadata/resources-3.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="https://api-platform.com/schema/metadata/resources-3.0 https://api-platform.com/schema/metadata/resources-3.0.xsd"
+>
+    <resource class="Sylius\Component\Core\Model\CatalogPromotion" shortName="CatalogPromotion">
+        <validationContext>
+            <values>
+                <value name="groups">
+                    <values>
+                        <value>sylius</value>
+                    </values>
+                </value>
+            </values>
+        </validationContext>
+
+        <operations>
+            <operation class="ApiPlatform\Metadata\Get"  uriTemplate="/admin/catalog-promotions/{code}.{_format}">
```

Comment:
New uriTemplate style `{code}.{_format}` wasn't in use in previous APIP3 PRs, please tell me where can I read about it?

I cannot find mentioned
https://api-platform.com/docs/core/content-negotiation/

Will `/admin/catalog-promotions/some-code` work or I will need to specify format each time, like `/admin/catalog-promotions/some-code.jsonld`?

Reply:
Generally, without using `_format` in the browser, I receive html (which is the default). However, if I add `_format`, I can display the response in `jsonld` by using  `/admin/catalog-promotions/some-code.jsonld`. Similary, for `json`, I can use `.json`.  I saw this in SymfonyCasts and [here](https://api-platform.com/docs/core/subresources/)

Reply:
We decided within the team that we're doing it without `_format`